Dockerfile

```diff
@@ -13,6 +13,11 @@ RUN npm install
 
 COPY . .
 
+RUN npm run compile
+COPY public.pem /build/
+COPY IP_DATA.bin /build/
+WORKDIR /build
```

Comment:
Why has the `WORKDIR` been changed to `/build`. Shouldn't the command `npm start` be executed in the directory where `package.json` is present?